.changes/unreleased/Fixes-20230630-092618.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Change batch_id to model override
+time: 2023-06-30T09:26:18.854492+01:00
+custom:
+  Author: nickozilla
+  Issue: "671"
```

Updating to resolve #822 (which I just created)

```suggestion
  Issue: "822"
```